django_rest_generator/utils.py

```diff
@@ -23,3 +23,49 @@ def sanitize_endpoint_to_method_name(endpoint):
     if endpoint.endswith("."):
         endpoint = endpoint[:-1]
     return endpoint
+
+
+def find_nested_keys(node, key):
+    if isinstance(node, list):
+        for i in node:
+            for x in find_nested_keys(i, key):
+                yield x
+    elif isinstance(node, dict):
+        if key in node:
+            yield node[key]
+        for j in node.values():
+            for x in find_nested_keys(j, key):
+                yield x
+
+
+def match_to_openapi_path_spec(openapi_path, request_path) -> bool:
+    """Match s against the given format string.
```

Comment:
you're comments arnt bad, but it would be nice to have an example in the docstrings for what the expected in and out look like, it would make it more clear to read